scripts/useful_commands.sh

```diff
@@ -29,4 +29,20 @@ function wait_next_block {
         break
     fi
   done
+}
+
+function wait_two_blocks {
```

Comment:
would it suffice to simply call `wait_next_block` twice?

actually, rename it to `wait_count_blocks` as follows:
```
function wait_count_blocks {
  for i in $(seq 1 $1); do
    wait_next_block
  done
}
```

Reply:
done. see https://github.com/lavanet/lava/commit/461d3dc2bac8029c3f2f3d7bb3664db2158c26e7